web-ui/package.json

```diff
@@ -17,11 +17,12 @@
     "@mui/material": "^5.15.14",
     "@mui/styled-engine-sc": "^6.0.0-alpha.18",
     "@mui/styles": "^5.15.14",
-    "@mui/x-date-pickers": "^7.0.0",
+    "@mui/x-date-pickers": "^7.3.1",
     "@tinymce/tinymce-react": "^5.0.0",
     "axios": "^1.6.8",
     "canvas-confetti": "^1.6.0",
     "date-fns": "^2.24.0",
+    "dayjs": "^1.11.11",
```

Comment:
Is this still needed or can we do everything we need with just the JavaScript `Date` class and the `date-fns` library that is already installed?

Reply:
I was able to remove this.